server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
wire is needed because of the need to share stuff there between the client and server. I still think its also nice to differentiate between the language and the server. I'm leaving it as is such the server will now be called sqlserver.Server. Its not bad.